src/route.js

```diff
@@ -93,12 +93,12 @@ module.exports = class Route {
 
       case Restypie.RouterTypes.KOA_ROUTER:
         /* istanbul ignore next */
-        return function * restypieRouteHandler (next) {
-          if (this.request.body) this.req.body = this.request.body;
-          this.req.params = this.params; // Copy params so that we don't have to parse them
-          this.req.query = this.query;
-          this.state.bundle = new Restypie.Bundle({ req: this.req, res: this.res, url });
-          yield next;
+        return async function restypieRouteHandler (ctx, next) {
```

Comment:
https://github.com/nodejs/LTS#lts-schedule1

node 4 will reach end of life 2018-04-01. also we should be using a luxe fork anyway.